services/file_get_contents.proto

```diff
@@ -1,13 +1,22 @@
+/**
+ * # File Get Contents
+ * Messages for a query to retrieve the content of a file in the
+ * Hedera File Service (HFS).
+ *
+ * ### Keywords
```

Comment:
IMHO this Keywords section at the top of every file is unnecessary and redundant. Could it be dropped or at least consolidated to a single overview 'about the documentation' file?